_posts/2025/2025-12-09-openloco-v25.12.md

```diff
@@ -139,6 +139,56 @@ Finally, the music selection window has been reworked by @LeeSpork. The window n
 music applies to, the list is now sortable, and resizeable! We hope this increases your playlist
 customisation prowess [quite a bit (#3384)](https://github.com/OpenLoco/OpenLoco/pull/3384).
 
+## Doubling down on CMake
+
+Over the past few years, we've continued to rely on CMake to tie our toolchain together. While initially
+we were dabbling with Visual Studio solution files for Windows, and several CMake files for Linux and even macOS,
+@Duncanspumpkin unified these to work on all platforms, and we're now reaping even more the fruits of this labour.
+You see, we had to rework our CMake toolchain again to make 64-bit builds possible.
+We took the opportunity to drop a bunch of now-unnecessary things in the process.
+Let's take them one at a time.
+
+### Interop removal
+
+First off, no longer needing to interop with the vanilla game has allowed us to
+[remove a fair bit of code (#3491)](https://github.com/OpenLoco/OpenLoco/pull/3491).
+This moment has been highly anticipated, and @Duncanspumpkin was all to eager to get it done.
+
+### CMake presets
+
+While CMake has been _the_ go-to tool to build OpenLoco for a while, it could be a little confusing
+to get it to build so-called _cross-platform_ builds, in this case building 32-bit software on a 64-bit system.
+Now that this is no longer necessary, CMake presets could be
+[simplified quite a bit (#3523)](https://github.com/OpenLoco/OpenLoco/pull/3523).
+The project readme was adjusted accordingly. Whether you're on Windows, Linux, macOS, or even BSD,
+it should be super easy to get set up the compiler toolchain now.
+
+### Linux builds
+
+For our _continous integration_ (CI) build platform, we generally rely on [vcpkg](https://github.com/microsoft/vcpkg)
+to build libraries we depend on, e.g. the SDL2 hardware abstraction layer.
+While Linux distributions tend to have their own package manager, you _can_ opt-in to using vcpkg
+as well if the system packages are not up-to-date enough.
+All this to say, our 'vcpkg' Linux preset has now been
+[updated to use x64 (#3494)](https://github.com/OpenLoco/OpenLoco/pull/3494) as well.
+
+### macOS builds
+
+Something part of the team has been eagerly anticipating has been the return of macOS builds.
+Even when the codebase wasn't _entirely_ ready, Aaron started to hack away at getting
+[native 64-bit builds (#3288)](https://github.com/OpenLoco/OpenLoco/pull/3288)
+for this playform. Much of the work tied in with the `loco_global` rework, and was ultimately split off
+into their own PRs to ease the review process.
+Duncan created a [CI job for macOS (#3499)](https://github.com/OpenLoco/OpenLoco/pull/3499)
+as well, so we now have regular macOS builds on GitHub again!
```

Comment:
Should probably mention the Gatekeeper issues here as well, in case anyone feels like contributing a fix we haven't thought of.